.vscode/launch.json

```diff
@@ -1,6 +1,7 @@
 {
   "version": "0.2.0",
   "configurations": [
+ 
```

Comment:
This change is not related to #6 or #9. I suggest don't make changes to `.vscode/launch.json`